autogpt_platform/frontend/src/hooks/useAgentGraph.tsx

```diff
@@ -476,7 +480,8 @@ export default function useAgentGraph(
         flowExecutionID,
       );
 
-      // Set graph execution error from the initial fetch
+      // Set graph execution status and error from the initial fetch
+      setGraphExecutionStatus(execution.status);
```

Comment:
This is yet 1 more state on this 1000-line hook. You sure there are no side-effect issues from this? 😆